parsons/load.py

```diff
@@ -27,19 +28,20 @@ def load_config_file(paths):
       pass
   raise Exception("Cannot find files {0}".format(paths))
 
-def load_config(file_name):
+def load_config(problem_name):
   """
-  Loads a YAML file, assuming that the YAML file is located in the problems/PROBLEM_NAME.yaml directory.
+  Loads a YAML file, assuming that the YAML file is located at {PROBLEM_PATHS}/{problem_name}.yaml
+  Normalizes problem_name to lowercase as all filenames should be lowercased.
+   
   Args:
-      file_name: The name of the directory in the data directory.
-      root_path: Optional argument that specifies the root_path for problems.
+      problem_name: The name of the problem.
 
   Returns: The contents of the YAML file as a defaultdict, returning None
       for unspecified attributes.
   """
   config_files = []
   for path in PROBLEM_PATHS:
-    config_files.append(os.path.join(os.path.abspath(path), file_name + ".yaml"))
+    config_files.append(os.path.join(os.path.abspath(path), problem_name.lower() + ".yaml"))
```

Comment:
This is the primary functionality change. The other changes are just documentation/style.